examples/nextjs-sdk/src/app/address/[address]/page.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @next/next/no-img-element */
+import { nameguard } from "@namehash/nameguard";
+import { Shield } from "../../components/Shield";
+
+export default async function Page({
+  params,
+}: {
+  params: { address: string };
+}) {
+  const { address } = params;
+
+  const data = await nameguard.getSecurePrimaryName(address);
+
+  return (
+    <div className="flex w-full flex-col h-screen overflow-hidden">
+      <div className="flex">
+        <div className="border-b border-gray-200 flex items-center px-2 md:px-4 py-3 border-l-0 z-10 max-md:h-fit md:max-h-sm w-full h-16">
+          <div className="flex w-full items-center">
+            <div className="mr-2 font-bold text-sm">To:</div>
+            <img
+              src={`https://effigy.im/a/${address}.png`}
```

Comment:
I've created a component that is `Avatar` to handle this logic. I've removed any mentions of `isLoadingPrimaryName` because this will always be `false` since we're server rendering these pages in Next App Router.